src/components/SingleName/RecordsItem.js

```diff
@@ -74,6 +72,25 @@ function chooseMutation(recordType) {
   }
 }
 
+const Actionable = ({
+  startEditing,
+  keyName,
+  value
+}) => {
+  if(value && value.error){
+    return(<></>)
```

Comment:
Just return null if you don't want to render anything. Alternatively we can just reverse it to `if(value && !value.error)` and drop the `else`